packages/cozy-procedures/src/redux/documentsDataSlice.js

```diff
@@ -0,0 +1,121 @@
+/*eslint no-unused-vars: off*/
+/*eslint no-console: off*/
+import { createSlice } from 'redux-starter-kit'
+import get from 'lodash/get'
+import creditApplicationTemplate from '../templates/creditApplicationTemplate'
+
+const documentsSlice = createSlice({
+  initialState: {
+    data: {
+      /*  payslip: {
+        ...files
+      } */
+    },
+    ui: {
+      /* payslip: {
+        loading: true,
+        error: false
+      } */
+    }
+  },
+  slice: 'documents',
+  reducers: {
+    init: (state, action) => {
+      return {
+        data: Object.keys(action.payload).reduce((acc, fieldId) => {
+          acc[fieldId] = {}
+          return acc
+        }, {}),
+        ui: Object.keys(action.payload).reduce((acc, fieldId) => {
+          acc[fieldId] = {
+            loading: false,
+            error: false
+          }
+          return acc
+        }, {})
+      }
+    },
+    update: (state, action) => {
```

Comment:
We can probably remove this, since we now have `fetchDocumentSuccess ` etc

Reply:
I think we'll use it when importing manually a doc ?

Reply:
Yeah I'm not sure. I think we can remove it as part of this PR so we don't feel forced to use it later on if it's not the right solution.